bigframes/_config/bigquery_options.py

```diff
@@ -265,20 +265,16 @@ def use_regional_endpoints(self) -> bool:
             in regions "europe-west3", "europe-west8", "europe-west9",
             "me-central2", "us-central1", "us-central2", "us-east1", "us-east4",
             "us-east5", "us-east7", "us-south1", "us-west1", "us-west2", "us-west3"
```

Comment:
Looks like we cut off a little too much.

```suggestion
            "us-east5", "us-east7", "us-south1", "us-west1", "us-west2", "us-west3"
            and "us-west4".
```

Reply:
Thanks for catch!